.github/ISSUE_TEMPLATE/nullness_issue.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Nullness-related report
+about: Create a report related to nullable reference types handling
+title: ''
+labels:  [Bug, Needs-Triage,Area-Nullness]
+assignees: 'T-Gro'
+
+---
+
+
+Please check at [Nullable Reference Types RFC](https://github.com/fsharp/fslang-design/blob/main/RFCs/FS-1060-nullable-reference-types.md) if this issue isn't a known limitation, such as missing flow-control analysis after branching constructs like `if` or `while`.
```

Comment:
Would be good to have a direct link to known issue later.

Reply:
These are orthogonal, will keep both.
Known limitations: Intentionaly not done, that is the RFC link
Known issue: Will point to a GH query over issues, i.e. already reported issues.